data/lib/src/network/linshare_http_client.dart

```diff
@@ -67,9 +69,18 @@ class LinShareHttpClient {
   }
 
   Future<bool> deletePermanentToken(PermanentToken token) async {
-    final deletedToken = await _dioClient.delete(
-      Endpoint.authentication.generateEndpointPathWithParameter(token.tokenId.uuid)
-    );
+    // final headerParam = _dioClient.getHeaders();
+    // headerParam.clear();
+    // headerParam[HttpHeaders.acceptHeader] = 'application/json';
+    // headerParam[HttpHeaders.contentTypeHeader] = 'application/json';
```

Comment:
I cleared but charset still there

Reply:
yes, that the point I want you work more on it. I can set up to execute the request succeeded. But it is not totally clear, Please check in the log I sent you yesterday:
```
I/flutter ( 3856): *** Request ***
I/flutter ( 3856): uri: https://user.linshare-2-3.integration-linshare.org/linshare/webservice/rest/user/v2/jwt/27caa887-f411-4cb9-8abd-2dab6ac0dd9d
I/flutter ( 3856): method: DELETE
I/flutter ( 3856): responseType: ResponseType.json
I/flutter ( 3856): followRedirects: true
I/flutter ( 3856): connectTimeout: 0
I/flutter ( 3856): receiveTimeout: 0
I/flutter ( 3856): extra: {}
I/flutter ( 3856): headers:
I/flutter ( 3856):  accept: application/json
I/flutter ( 3856):  content-type: application/json; charset=utf-8
I/flutter ( 3856):  Content-type: application/json
I/flutter ( 3856):  Cookie: JSESSIONID=218F28F6549A67CFA675A2E4688F3A26
I/flutter ( 3856): data:
I/flutter ( 3856): null
I/flutter ( 3856): 
I/flutter ( 3856): *** Response ***
I/flutter ( 3856): uri: https://user.linshare-2-3.integration-linshare.org/linshare/webservice/rest/user/v2/jwt/27caa887-f411-4cb9-8abd-2dab6ac0dd9d
I/flutter ( 3856): statusCode: 200
I/flutter ( 3856): headers:
I/flutter ( 3856):  access-control-allow-headers: WWW-No-Authenticate
I/flutter ( 3856):  cache-control: max-age=0,no-cache,no-store
I/flutter ( 3856):  date: Wed, 09 Dec 2020 10:40:09 GMT
I/flutter ( 3856):  content-length: 320
I/flutter ( 3856):  content-type: application/json
I/flutter ( 3856):  x-frame-options: DENY
I/flutter ( 3856):  server: Apache
I/flutter ( 3856):
```
Two different `content-type`, so I want only 1

Reply:
I think `deleteToken` work with `Content-Type`
please check here:
```
  void _provideBaseOption() {
    final headers = <String, dynamic>{
      HttpHeaders.acceptHeader: 'application/json',
      'Content-Type': 'application/json'
    };
    getIt.registerSingleton<BaseOptions>(BaseOptions(headers: headers));
  }
```
with log:
```
I/flutter ( 2969): *** Request ***
I/flutter ( 2969): uri: https://user.linshare-2-3.integration-linshare.org/linshare/webservice/rest/user/v2/jwt/51c561b5-0561-42c4-9bfe-ec63c46d7aaa
I/flutter ( 2969): method: DELETE
I/flutter ( 2969): responseType: ResponseType.json
I/flutter ( 2969): followRedirects: true
I/flutter ( 2969): connectTimeout: 0
I/flutter ( 2969): receiveTimeout: 0
I/flutter ( 2969): extra: {}
I/flutter ( 2969): headers:
I/flutter ( 2969):  accept: application/json
I/flutter ( 2969):  Content-Type: application/json
I/flutter ( 2969):  content-type: application/json; charset=utf-8
I/flutter ( 2969):  Cookie: JSESSIONID=AA8BC710C490D19506BDF42E39EDDFD6
I/flutter ( 2969): data:
I/flutter ( 2969): null
I/flutter ( 2969): 
I/flutter ( 2969): *** Response ***
I/flutter ( 2969): uri: https://user.linshare-2-3.integration-linshare.org/linshare/webservice/rest/user/v2/jwt/51c561b5-0561-42c4-9bfe-ec63c46d7aaa
I/flutter ( 2969): statusCode: 200
I/flutter ( 2969): headers:
I/flutter ( 2969):  access-control-allow-headers: WWW-No-Authenticate
I/flutter ( 2969):  cache-control: max-age=0,no-cache,no-store
I/flutter ( 2969):  date: Wed, 09 Dec 2020 22:10:26 GMT
I/flutter ( 2969):  content-length: 320
I/flutter ( 2969):  content-type: application/json
I/flutter ( 2969):  x-frame-options: DENY
I/flutter ( 2969):  server: Apache
```